src/app/main.rs

```diff
@@ -663,7 +692,7 @@ where
             // implementations can use the route-specific configuration.
             let dst_route_stack = svc::builder()
                 .layer(buffer::layer(max_in_flight))
-                .layer(pending::layer())
+                .layer(pending::layer::<_, dst::Route, _>())
```

Comment:
This type annotation is necessary as a result of changing the `pending::layer` function to bound the target type.